javascript-data-model/order-history.js

```diff
@@ -0,0 +1,55 @@
+var orderHistory = [
+  {
+    ordered: 'August 4, 2020',
+    total: '$34.00',
+    ship: 'JS Masher',
+    delivered: 'Delivered Aug 8, 2020',
+    deliveryLocation: 'Your package was delivered.It was handed directly to a resident.',
+    orderNumber: '114-3941689-8772232',
+    title: 'JavaScript for Impatient programmers',
+    author: 'Rauschmayer, Dr. Axel',
+    returnBy: 'Return window Closed on Sep 7, 2020',
+    retailPrice: '$31.55',
+    imgage: 'images/images1.png'
+  },
```

Comment:
All the data for the item in the order is intermixed with the data about the order itself, which is confusing. You should organize the data for each item and store it into a property on the object representing the order.